javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
+test-only resources when experimenting with these examples.
+
+To run these examples, you can setup your development environment to use Apache Maven or Gradle to configure and build
+AWS SDK for Java projects. For more information,
+see [Get started with the AWS SDK for Java 2.x](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html)
+.
+
+## Testing the Amazon CloudFront files
+
+You can test the Java code examples with two JUnit5 test classes.
+
+The `CloudFrontTest` class tests the following files:
+
+| Java file          | `config.properties` entry required | Comment                                                                       |
+|--------------------|------------------------------------|-------------------------------------------------------------------------------|
+| CreateFunction     |                                    | Uses provided `CF_function.js` file                                           |
+| DescribeFunction   |                                    |                                                                               |
+| DeleteFunction     |                                    |                                                                               |
+| ListFunction       |                                    |                                                                               |
+| GetDistribution    |                                    |                                                                               |
+| ModifyDistribution | `distributionId`                   | This is the distributionId of an existing distribution, e.g., `EHUCVLZEPS70R` |
+
+The `CloudFrontSigningTest` class tests the following files:
+
+| Java file                   | `config.properties` entry required | Comment                                                                                                                                                                                                                                                                                                                                           |
+|-----------------------------|------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CreatePublicKey             | `publicKeyFileName`                | Put a copy of your public key file in `src/main/resources` directory. <br/>The public key is the counterpart to the private key you use to sign.<br/>See the [CloudFront Developer Guide](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs). |
```

Comment:
in `src/main/resources` directory. --> in the `src/main/resources` directory.

See the --> For more information, see the